server/src/test/java/org/candlepin/dto/api/v1/ProductTranslatorTest.java

```diff
@@ -156,4 +160,25 @@ protected void verifyOutput(Product source, ProductDTO dto, boolean childrenGene
             assertNull(dto);
         }
     }
+
+    /**
+     * Verifies that the product's sets of products are translated properly.
+     *
+     * @param originalProducts the original collection of products we check against.
+     * @param dtoProducts      the translated DTO set of products that we need to verify.
```

Comment:
Generally try to avoid this type of code or comment formatting. It makes it very difficult to maintain as the code changes, since it will require constant reformatting to maintain the proper spacing. If you want to keep things aligned for parameters, use the line break + indent method:

```
/**
 * @param myParam
 *   Param details here
 **/
```

Reply:
Whole codebase is filled with random type of formatting for method description. Will follow above type of formatting onwards. Changes made for whole PR.

Reply:
You are not wrong in our inconsistency there. For the most part it's not enough of a bother to mention, but this one style in particular is a maintenance problem, due to the variable whitespace required depending on variable-length labels.

If you prefer the inline style, I'm not a huge fan of it myself, but I don't bother commenting on it if the inlined whitespace is absent.